pandas/tests/generic/test_generic.py

```diff
@@ -202,6 +202,12 @@ def test_downcast(self):
         expected = o.astype(np.int64)
         self._compare(result, expected)
 
+    def test_multiple_astype_casts(self):
+        df = DataFrame({'A': [1, 2, 'z'], 'B': [3, 4, 'z']})
+        df.astype({'A': int,
+                   'B': 'datetime64[ns]'},
+                  errors='ignore')
+
```

Comment:
If you can also add the example #25905, we can close that out.

Reply:
Will do.

Reply:
Added in d7c9d5a350ba9e9fd79e0300a5867911d306c600